allennlp/predictors/wikitables_parser.py

```diff
@@ -59,12 +22,9 @@ def _json_to_instance(self, json_dict: JsonDict) -> Instance:
         question_text = json_dict["question"]
         table_rows = json_dict["table"].split('\n')
 
-        # pylint: disable=protected-access
-        tokenized_question = self._dataset_reader._tokenizer.tokenize(question_text.lower())  # type: ignore
         # pylint: enable=protected-access
         instance = self._dataset_reader.text_to_instance(question_text,  # type: ignore
-                                                         table_rows,
-                                                         tokenized_question=tokenized_question)
+                                                         table_rows)
```

Comment:
Does this need additional processing?  Does this have to be formatted as a corenlp-tagged table?

Reply:
It is just the raw table content. I added a comment here and updated the docstring of `text_to_instance` to make that clear.